### PR TITLE
Display dataset recommendations from udata-recommendations

### PR DESCRIPTION
**Pull request by abulte**
Related to https://github.com/etalab/udata-gouvfr/pull/294 and https://github.com/opendatateam/udata-recommendations/pull/1/files

The `template_hook` thingy might not be ideal, RFC.

### CODE REVIEW

File: udata/models/extras_fields.py [hunk 1]
```diff
@@ -66,3 +66,6 @@ def validate(self, value):
             types = ', '.join(t.__name__ for t in ALLOWED_TYPES)
             raise ValidationError('Value should be an instance of: {types}',
                                   types=types)
+        if isinstance(value, list):
```
Comment (noirbizarre) [inline udata/models/extras_fields.py]: Maybe `isinstance(value, (list, tuple))`

File: udata/frontend/__init__.py [hunk 2]
```diff
@@ -21,13 +21,24 @@
...
 
 
+def template_hook(func):
+    _template_hooks[func.__name__] = func
+    return func
+
+
+@front.app_context_processor
+def inject_template_hooks():
+    return {'hook_%s' % k: v for (k, v) in _template_hooks.iteritems()}
```
Comment (noirbizarre) [inline udata/frontend/__init__.py]: Maybe prepare for Python 3, use `items()` (We won't have a lot of hooks, difference won't be noticeable)

File: udata/models/extras_fields.py (hunk 1 above)
Comment (abulte) [reply udata/models/extras_fields.py]: 👍

File: udata/frontend/__init__.py (hunk 2 above)
Comment (abulte) [reply udata/frontend/__init__.py]: 👍

**Pull request merged**